neo/VM/Helper.cs

```diff
@@ -228,5 +228,67 @@ private static ContractParameter ToParameter(StackItem item, List<Tuple<StackIte
             }
             return parameter;
         }
+
+        public static StackItem ToStackItem(this ContractParameter parameter)
+        {
+            return ToStackItem(parameter, null);
+        }
+
+        private static StackItem ToStackItem(ContractParameter parameter, List<Tuple<StackItem, ContractParameter>> context)
+        {
```

Comment:
We need to check the recursion as in `Serialize`?

Reply:
The context is used to check recursion, but if the parameter is deserialized from JSON the recursion may not be met.